### PR TITLE
Add fetch() into index.html and script.js

### PR DESCRIPTION
**Pull request by emilyzhaoo**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -28,3 +28,10 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+// Add fetch data
```
Comment (EkkiD) [inline portfolio/src/main/webapp/script.js]: This comment needs a bit more detail to help the reader understand. What is the fetch data? Where are we adding it to?

**Review by EkkiD — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (emilyzhaoo) [reply portfolio/src/main/webapp/script.js]: Sounds good, will change!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -28,3 +28,12 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+/**  
+Fetch data from the server using the async and await keywords and adds it to the DOM.
+*/
+async function getResponse() {
+  const response = await fetch('/data');
+  const quote = await response.text();
```
Comment (azhang72) [inline portfolio/src/main/webapp/script.js:37]: Use COSNTANT_CASE for constants (both RESPONSE and QUOTE).

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -28,3 +28,12 @@ function addRandomFact() {
...
+  document.getElementById('quote-container').innerText = quote;
+}
```
Comment (azhang72) [inline portfolio/src/main/webapp/script.js:39]: nit: add new line at end of file.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -28,3 +28,12 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+/**  
+Fetch data from the server using the async and await keywords and adds it to the DOM.
+*/
+async function getResponse() {
```
Comment (EkkiD) [inline portfolio/src/main/webapp/script.js:35]: nit: don't need to specify async/await, but otherwise the coment looks good.

**Review by EkkiD — APPROVED** [review]
LGTM

**Pull request merged**